chainio/clients/builder.go

```diff
@@ -45,7 +44,7 @@ type Clients struct {
 	PrometheusRegistry     *prometheus.Registry  // Used if avs teams need to register avs-specific metrics
 }
 
-func BuildClients(config Config, logger logging.Logger) (*Clients, error) {
+func BuildAll(config BuildAllConfig, logger logging.Logger) (*Clients, error) {
```

Comment:
I think builder should also accept signer. this only supports privatekeyhex which is good for testing but what about bulding all with  the local keystore or remote signer etc? unless you are only intending this for tests?

Reply:
good call. fixed in [32ca000](https://github.com/Layr-Labs/eigensdk-go/pull/73/commits/32ca0003afdcd450f7121dc54889238907c19abc)